src/services/completions.ts

```diff
@@ -564,7 +576,11 @@ namespace ts.Completions {
                 };
             }
 
-            return undefined;
+            return {
+                isMemberCompletion: false,
+                isNewIdentifierLocation: false,

```

Comment:
The flag should be true in both cases.

`isNewIdentifierLocation` is used to tell the editor that the user may be typing in an identifier that doesn't yet exist anywhere in the language service's context. So if you are typing in a new variable, eg `var fo|` (cursor at |), the LS has no idea how you might finish the statement.

Editors use this to change their behavior in terms of how they encourage you to choose a completion or not. In VS, for example, the behavior of pressing Enter changes depending on whether the flag is set (if true, pressing enter inserts a newline and doesn't take a highlighted completion. If false, pressing enter inserts the selected completion. Behavior when pressing Tab is unaffected by the flag -- the completion is always inserted).

In our case, someone writing a triple slash reference path may want to point to a file that doesn't yet exist, so we should allow them to create what is potentially a "new identifier" in this case.

I've refactored the code to reflect this observation, and added a comment.